### PR TITLE
[16.0][MIG] web_pivot_computed_measure: Migration to 16.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT44372

ping @chienandalu @pilarvargas-tecnativa

### CODE REVIEW

**Comment by pilarvargas-tecnativa** [comment]
Please check the tests, don't find odoo_test_helper

File: web_pivot_computed_measure/static/src/test/test.esm.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /** @odoo-module **/
-/* Copyright 2022 Tecnativa - Carlos Roca
+/* Copyright 2022-2024 Tecnativa - Carlos Roca
```
Comment (chienandalu) [inline web_pivot_computed_measure/static/src/test/test.esm.js]: There's no need to tweak this, copyright is understood to be from the stated year to any present time

**Review by chienandalu — APPROVED** [review]
Aside from the CI issues. Test and code review :+1:

**Comment by CarlosRoca13** [comment]
Changes done, and ci should be solved :)

**Comment by CarlosRoca13** [comment]
UPS! :sweat_smile:

**Comment by CarlosRoca13** [comment]
Now it is ready!

File: web_pivot_computed_measure/readme/ROADMAP.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#. Add support to define a style for a computed measure (ex. colored)
+#. Use t-model to data-binding instad of jquery selectors
```
Comment (pilarvargas-tecnativa) [inline web_pivot_computed_measure/readme/ROADMAP.rst]:
```suggestion
#. Use t-model to data-binding instead of jquery selectors
```

**Review by pilarvargas-tecnativa — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
Changed! @pilarvargas-tecnativa

**Comment by CarlosRoca13** [comment]
ping @pedrobaeza

**Pull request merged**